src/components/map.js

```diff
@@ -63,7 +63,10 @@ let Map = class Map extends React.Component {
       zoom: 3.5,
     });
 
+    var controls = new mapboxgl.NavigationControl();
```

Comment:
Could you [disable the compass button](https://docs.mapbox.com/mapbox-gl-js/api/markers/#navigationcontrol) here?